tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -118,6 +118,20 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
       aiLabPage = new AILabPage(page, webview);
       await aiLabPage.navigationBar.waitForLoad();
     });
+    test('Get API version', async ({ request, navigationBar }) => {
```

Comment:
you still can create a describe section, that will test opening of the  Local Server tab, reading the port number -> which you should still read dynamically, not have it hardcoded, then check that status of the server (OK) . Then check version and compare. This can be two test cases.